internal/framework/service/dcv_delegation/data_source.go

```diff
@@ -0,0 +1,66 @@
+package dcv_delegation
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/cloudflare/terraform-provider-cloudflare/internal/framework/muxclient"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var _ datasource.DataSource = &CloudflareDCVDelegationDataSource{}
```

Comment:
we can drop the `Cloudflare` prefix here.

```suggestion
var _ datasource.DataSource = &DCVDelegationDataSource{}
```